.github/workflows/codecov.yaml

```diff
@@ -0,0 +1,24 @@
+name: Codecov
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+permissions: read-all
```

Comment:
There aren't any repo tokens that a user running from a fork could get right?

Reply:
No, there aren't!